content/events/the-cofhe-shop.md

```diff
@@ -0,0 +1,37 @@
+---
+eventName: "The CoFHE Shop" #the name of your track or event and its mandatory
+
+eventDescription: "The CoFHE Shop is an initiative by Zama to create a dynamic space for the Fully Homomorphic Encryption (FHE) community during EthCC [7] week. It offers opportunities to network, hack, enjoy good coffee, and learn about FHE" #short description of your track or event limiting to 100-150 characters
```

Comment:
@laeccles Please confirm if it's 'EthCC [7] week' or just 'EthCC week'.

Reply:
@WinsCodes Please remove [7].

Thanks!